suite/tests/client-interface/memory_dump_test.expect

```diff
@@ -0,0 +1,3 @@
+thank you for testing memory dump
+thread init
+Hello, world!
```

Comment:
This test needs to be enabled in suite/tests/CMakeLists.txt.

How did you test it?  It won't even be built, it looks like.  Please test and ensure it runs in the suite.

Reply:
$ !ctest
ctest -R memory_dump
Test project /usr/local/google/home/kyluk/dynamorio-x86/dynamorio/build
    Start 186: code_api|client.memory_dump_test
1/1 Test #186: code_api|client.memory_dump_test ...   Passed    0.05 sec

I forgot to add the file suite/tests/CMakeLists.txt in the commit. Here's the diff

$ git diff suite/tests/CMakeLists.txt
diff --git a/suite/tests/CMakeLists.txt b/suite/tests/CMakeLists.txt
index 5bb52650e..b6b001679 100644
--- a/suite/tests/CMakeLists.txt
+++ b/suite/tests/CMakeLists.txt
@@ -2646,6 +2646,9 @@ if (NOT ANDROID) # TODO i#38: Port test to Android.
     torunonly_ci(client.attach_blocking linux.infloop client.attach_test.dll
       client-interface/attach_blocking.runall "" "" "")
   endif ()
+  if (LINUX AND X64)
+    tobuild_ci(client.memory_dump_test client-interface/memory_dump_test.c "" "" "")
+  endif ()
 endif ()
 
 if (UNIX)